### ISSUE TITLE
Failed build on Crystal 1.10

### ISSUE THREAD

**Issue by zw963** [bcardiff/crystal-repl-server#1]
```
 ╰─ $ make
mkdir -p ./bin
CRYSTAL_CONFIG_PATH="lib:/home/zw963/Crystal/bin/../share/crystal/src" CRYSTAL_CONFIG_LIBRARY_PATH="/home/zw963/Crystal/bin/../lib/crystal" crystal build  src/server/cli.cr -o ./bin/crystal-repl-server
Showing last frame. Use --error-trace for full trace.

In /home/zw963/Crystal/share/crystal/src/compiler/crystal/compiler.cr:147:35

 147 | property dependency_printer : DependencyPrinter? = nil
                                     ^----------------
Error: undefined constant DependencyPrinter
make: *** [Makefile:16: bin/crystal-repl-server] Error 1
```

Thanks

**Comment by bcardiff** [comment]
Closed in 413bfce4ef9ea02d7961c9c4c937fc1bd673cdfd

Are you successfully using the crystal-repl-server for something?

**Comment by zw963** [comment]
It works.

> Are you successfully using the crystal-repl-server for something?

Still not yet, sorry, just try like a toy for now.
